test/oneNoteDataStructures/oneNoteItemUtils.spec.ts

```diff
@@ -1,9 +1,93 @@
 import { Notebook } from '../../src/oneNoteDataStructures/notebook';
 import { OneNoteItem } from '../../src/oneNoteDataStructures/oneNoteItem';
-import { OneNoteItemUtils } from '../../src/oneNoteDataStructures/oneNoteItemUtils';
+import { SectionGroup } from '../../src/oneNoteDataStructures/sectionGroup';
+import { OneNoteItemUtils, SectionPathElement } from '../../src/oneNoteDataStructures/oneNoteItemUtils';
 import { Section } from '../../src/oneNoteDataStructures/section';
 
 describe('OneNoteItemUtils', () => {
+	const notebook1: Notebook = {
```

Comment:
You have a number of entities here that your test use throughout several tests here. I'll simplify a bit:
```
const notebook1: Notebook = {
... sections: [] ...
}
it('test1', () => {
        notebook1.push(section1);
		const depthOfNotebooks = OneNoteItemUtils.getDepthOfNotebooks([notebook1]);
		expect(depthOfNotebooks).toEqual(2);
	});
it('test2', () => {
        const depthOfNotebooks = OneNoteItemUtils.getDepthOfNotebooks([notebook1]);
		expect(depthOfNotebooks).toEqual(2);
	});

```

If both tests run in the same persisted browser instance, then the order in which the tests run affects whether they pass or not, because they both use notebook1. A better way to build this could be:
```
function createNewNotebook(){
  return new {
    ... sections: [] ...
  }
}
it('test1', () => {
        var notebookMock = createNotebookMock();
        notebook1.push(section1);
		const depthOfNotebooks = OneNoteItemUtils.getDepthOfNotebooks([notebookMock]);
		expect(depthOfNotebooks).toEqual(2);
	});
it('test2', () => {
        const depthOfNotebooks = OneNoteItemUtils.getDepthOfNotebooks([createNotebookMock()]);
		expect(depthOfNotebooks).toEqual(2);
	});

```

Reply:
You can also have a parameter to specify the notebook name (if you wish to) in createNewNotebook().